### PR TITLE
docs: add EN readme for noto sans font

### PR DESCRIPTION
**Pull request by melaniebmn**
# Summary | Résumé

Adding the EN readme for the Noto Sans font. I will add the FR content once it comes back from translation.

### CODE REVIEW

File: fonts/noto-sans/README.md [hunk 1]
```diff
@@ -1,3 +1,211 @@
...
+
+### Install `Noto Sans` font with npm
+
+Navigate to the root folder of your project and run:
+
+```js
+npm install @cdssnc/gcds-fonts
+```
+
+Place the following code in your CSS and replace `path/to/node_modules` with the location where you've added the node modules:
```
Comment (daine) [inline fonts/noto-sans/README.md:118]:
```suggestion
Place the following code in your CSS and replace `path/to/file` with the location where you've added the node modules:
```

I can't see `path/to/node_modules` in the code below, maybe it's supposed to be `path/to/file`? Or the other way around works too!

**Review by daine — CHANGES_REQUESTED** [review]
Just one tiny inconsistency!

File: fonts/noto-sans/README.md (hunk 1 above)
Comment (melaniebmn) [reply fonts/noto-sans/README.md:118]: Good catch, updated the paths - thank you!

**Review by daine — APPROVED** [review]
LGTM!

**Pull request merged**